creation/web_base/condor_startup.sh

```diff
@@ -656,6 +659,9 @@ GLIDEIN_VARIABLES = $glidein_variables
 MASTER_NAME = glidein_${glidein_startup_pid}_${random_name_str}
 STARTD_NAME = glidein_${glidein_startup_pid}_${random_name_str}
 
+#Stats that make detection of black-hole slots
+STARTD.STATISTICS_TO_PUBLISH_LIST = \$(STATISTICS_TO_PUBLISH_LIST) JobBusyTime JobDuration
+
```

Comment:
Minor. These lines could go in the common section below (GLIDEIN_BLACKHOLE_NUMJOBS = $glidein_blackhole_numjobs...) so it's all together.

Reply:
that's a good idea!